### PR TITLE
update dashboards content for GA

### PR DESCRIPTION
**Pull request by imatwawana**
Completing initial updates to structure of Dashboards content for GA.

### CODE REVIEW

File: src/docs/product/dashboards/customize-dashboards/index.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+description: "Learn how to edit or create additional dashboards to suit your organization's needs."
+---
+
+<Note>
+
+Editing dashboards is only available if your organization is on either the Business plan or the Trial plan.
+
+</Note>
+
+When you visit Dashboards for the first time, you'll find a dashboard with a set of pre-built widgets. You can build a new dashboard or edit an existing dashboard by clicking on either the "Create Dashboard" or the "Edit Dashboard" buttons on the top right-hand corner.
```
Comment (PeloWriter) [inline src/docs/product/dashboards/customize-dashboards/index.mdx:13]: a thought: if either of these buttons move, the content won't be evergreen. If they're already pretty self-evident, we don't need to tell users to look in the top RH corner.

```suggestion
When you visit Dashboards for the first time, you'll find a dashboard with a set of pre-built widgets. You can build a new dashboard or edit an existing dashboard by clicking on either "Create Dashboard" or "Edit Dashboard" on the top right-hand corner.
```

File: src/docs/product/dashboards/customize-dashboards/index.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Next, you can build or edit widgets by:
+
+- Clicking on the pencil icon to edit an existing widget.
+- Clicking on the widget panel with the plus icon to create a new widget.
```
Comment (PeloWriter) [inline src/docs/product/dashboards/customize-dashboards/index.mdx:18]: suggest rephrasing to focus on what you can do, so:
```suggestion
- Edit an existing widget by clicking the pencil.
- Create a new widget by clicking the widget panel then with the plus icon.
```

File: src/docs/product/dashboards/customize-dashboards/index.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+<Note>
+
+Editing dashboards is only available if your organization is on either the Business plan or the Trial plan.
+
+</Note>
+
+When you visit Dashboards for the first time, you'll find a dashboard with a set of pre-built widgets. You can build a new dashboard or edit an existing dashboard by clicking on either the "Create Dashboard" or the "Edit Dashboard" buttons on the top right-hand corner.
+
+Next, you can build or edit widgets by:
```
Comment (PeloWriter) [inline src/docs/product/dashboards/customize-dashboards/index.mdx:15]:
```suggestion
Next, you can build or edit widgets:
```

File: src/docs/product/dashboards/filter-dashboards/index.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Filter Dashboards
+sidebar_order:
+description: "Learn more about filtering options in Dashboards."
+---
+
+Once you've created a dashboard, you can filter the information displayed in different ways.
+
+## Filter by Global Selection Header
```
Comment (PeloWriter) [inline src/docs/product/dashboards/filter-dashboards/index.mdx:9]:
```suggestion
## Global Selection Header
```

File: src/docs/product/dashboards/filter-dashboards/index.mdx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+You can specify which projects, environments, and date range you want to zoom in on at the top of the page. This can also be found in other parts of [sentry.io](https://sentry.io) as a top-level filter.
+
+## Filter by Interactive Graph
```
Comment (PeloWriter) [inline src/docs/product/dashboards/filter-dashboards/index.mdx:13]:
```suggestion
## Interactive Graph
```

File: src/docs/product/dashboards/index.mdx [hunk 6]
```diff
@@ -8,90 +8,18 @@ redirect_from:
...
 ---
 
-<Alert title="Note" level="info">
-  Dashboards are only available if you're in the Early Adopter program. Features available to Early Adopters are still in progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
-</Alert>
+<Note>
 
-Dashboards provide the ability to build custom views of your application's health, including the ability to navigate issues and performance across multiple projects.
-Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
+Dashboards are only available if you're in the Early Adopter program. Features available to Early Adopters are still in progress and may have bugs. We recognize the irony.
```
Comment (PeloWriter) [inline src/docs/product/dashboards/index.mdx:13]:
```suggestion
Dashboards are available only if you're in the Early Adopter program. Features available to Early Adopters are still in progress and may have bugs. We recognize the irony.
```

File: src/docs/product/dashboards/customize-dashboards/index.mdx (hunk 1 above)
Comment (imatwawana) [reply src/docs/product/dashboards/customize-dashboards/index.mdx:13]: Based on the screenshots, the buttons are pretty self-evident, so I've implemented your suggestion in your suggested version.

File: src/docs/product/dashboards/index.mdx [hunk 7]
```diff
@@ -8,90 +8,21 @@ redirect_from:
...
-found in other parts of Sentry as a top-level filter.
+You can filter the information that's displayed in a dashboard in different ways:
 
-## Filter by Interactive Graph
+- Global Selection Header: Specify which projects, environments, and date range you want to zoom in on at the top of the page. This can also be found in other parts of [sentry.io](https://sentry.io) as a top-level filter.
+- Interactive Graphs: Use the interactive graph function of time series visualizations to zoom in. To do this, click and drag over the area you want to investigate.
 
-Time Series visualizations have an interactive graph that allows you to zoom in.
-To zoom in, simply click and drag over the area you want to investigate.
+Dashboards are shared across users of your organization, and you can customize them, as required. For more information, see [Customize Dashboards](/product/dashboards/customize-dashboards/index/).
```
Comment (imatwawana) [inline src/docs/product/dashboards/index.mdx:28]:
```suggestion
Dashboards are shared across users of your organization, and you can customize them, as required. For more information, see [Customize Dashboards](/product/dashboards/customize-dashboards/).
```

**Review by doralchan — APPROVED** [review]

**Pull request merged**